sites/jock2go_eu/ad_parser.py

```diff
@@ -0,0 +1,126 @@
+        label_str = self.soup.select_one(".profileContactLabel")
+        if label_str:
+            phone_val = label_str.find_next_sibling("span")
+            if phone_val:
+                return phone_val.string.strip()
+        return None
+
+    def phone_numbers(self):
+        return [self.primary_phone_number()]
+
+    def date_posted(self):
+        return None
+
+    def name(self):
+        name_div = self.soup.select_one(".profile-contact-username")
+        if name_div:
+            return name_div.string.strip()
+        return None
+
+    def primary_email(self):
+        return None
+
+    def emails(self):
+        return []
+
+    def social(self):
+        return None
+
+    def age(self):
+        age_label = self.soup.find("div", class_="left", string="Age")
+        age_value = age_label.find_previous("div", class_="right")
+
+        if age_value:
+            return age_value.string.strip()
+        return None
+
+    def image_urls(self):
+        urls = []
+
+        img_links = self.soup.select("a.galleryBoxImage")
+
+        for img_link in img_links:
+            style = img_link.get_attribute_list("style")[0]
+            if not style is None:
+                url = style.replace("background-image: url('", ""). \
+                            replace("');", ""). \
+                            replace("thumb", "main")
+                if url.startswith("//"):
+                    url = "https:" + url
+                urls.append(url)
+
+        return urls
+
+    def location(self):
+        location_label = self.soup.select_one("a.profile-contact-location")
+        if location_label:
+            return location_label.string.strip()
+
+        return None
+
+    def ethnicity(self):
+        ethnicity_label = self.soup.find("div", class_="left", string="Ethnicity")
+        ethnicity_value = ethnicity_label.find_previous("div", class_="right")
+        if ethnicity_value:
+            return ethnicity_value.string.strip()
+        return None
+
+    def gender(self):
+        gender = self.provider_obj("gender")
+        if not gender:
+            return "Male"
+        return gender
+
+    def services(self):
+        return []
+
+    def website(self):
+        return None
+
+    def ad_text(self):
+        profile_desc = self.soup.select_one("div.profile-desc")
+        if profile_desc:
+            return profile_desc.get_text().strip()
+
+        return None
+
+    def ad_title(self):
+        ad_title_div = self.soup.select_one("h2.profile-inner-title")
+        if ad_title_div:
+            return ad_title_div.get_text().strip()
+
+        return None
+
+    def orientation(self):
+        orientation_label = self.soup.find("div", class_="left", string="Orientation")
+        orientation_value = orientation_label.find_previous("div")
+        if orientation_value:
+            return orientation_value.string.strip()
+        return None
+
+
+    def provider_obj(self, target_key):
```

Comment:
```suggestion
    def _provider_obj(self, target_key):
```
This is a convention used elsewhere in the repo to indicate that the function is not used external to this file.